dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -530,6 +530,11 @@
 			<!-- Suppress trimmer warnings unless we're trimming all assemblies -->
 			<SuppressTrimAnalysisWarnings Condition="'$(SuppressTrimAnalysisWarnings)' == '' And '$(TrimMode)' == 'full'">false</SuppressTrimAnalysisWarnings>
 			<SuppressTrimAnalysisWarnings Condition="'$(SuppressTrimAnalysisWarnings)' == ''">true</SuppressTrimAnalysisWarnings>
```

Comment:
I think this is a no-op because of the settings in Xamarin.Shared.Sdk.props. Should we remove it, @rolfbjarne?

Reply:
Yes, it looks like all the SuppressTrimAnalysisWarnings property setters here are no-ops, because SuppressTrimAnalysisWarnings shouldn't be empty at this point.

Then below, add the _ExtraTrimmerArgs like this:

```xml
<_ExtraTrimmerArgs Condition="'$(SuppressTrimAnalysisWarnings)' == 'true' Or '$(_UseNativeAot)' == 'true'">$(_ExtraTrimmerArgs) --notrimwarn</_ExtraTrimmerArgs>
```

This way we'll:

* Suppress ILLink's trimmer warnings when using NativeAOT
* Honor SuppressTrimAnalysisWarnings otherwise.